packages/editor/src/panels/viewport/index.tsx

```diff
@@ -168,11 +168,9 @@ function ViewportContainer() {
         {sceneName.value ? <SelectionBox viewportRef={ref} toolbarRef={toolbarRef} /> : null}
         {sceneName.value ? <TransformGizmoTool /> : null}
         {sceneName.value ? <CameraGizmoTool viewportRef={ref} toolbarRef={toolbarRef} /> : null}
+        <div id="engine-renderer-canvas-container" ref={ref} className="absolute h-full w-full" />
```

Comment:
does this still show the logo correctly when no scene is loaded?

Reply:
yes